### ISSUE TITLE
ClrAriaLiveService doesn't announce from wizard

### ISSUE THREAD

**Issue by tovinayv** [vmware/clarity#5083]
## Describe the bug

ClrAriaLiveService doesn't announce from wizard

## How to reproduce

https://stackblitz.com/edit/clarity-v4-dark-theme-wizard

1.  Enable VoiceOver Accessibility (Mac os, chrome)
2. Go to https://clarity-v4-dark-theme-wizard.stackblitz.io/
3. Click on 'Show Wizard Button'
4. Click 'Show Loading' Button
5.  See spinner appears but it doesn't announce 'Loading'.

## Expected behavior

When this.clrAriaLiveService.announce( "Loading",  ClrAriaLivePoliteness.assertive) invoked, Screen reader should announce "Loading"

## Versions

**App**

- Angular: 10
- Clarity: 4

**Device:**

- Type: MacBook
- OS: MacOS Mojave 10.14.6
- Browser Chrome 
- Version 85

## Additional notes
```
<clr-wizard #wizard [(clrWizardOpen)]="opened">

	<clr-wizard-title>Clarity Wizard Demo</clr-wizard-title>

	<clr-wizard-button [type]="'cancel'">Cancel</clr-wizard-button>
	<clr-wizard-button [type]="'previous'">Back</clr-wizard-button>
	<clr-wizard-button [type]="'next'">Next</clr-wizard-button>
	<clr-wizard-button [type]="'finish'">Finish</clr-wizard-button>

	<clr-wizard-page>
		<ng-template clrPageTitle>Page 1</ng-template>
		<button (click)="toggleLoading()" class="btn">{{loading ? 'Hide' : 'Show'}} Loading</button>
		<clr-spinner *ngIf="loading">
			Loading
		</clr-spinner>
	</clr-wizard-page>


</clr-wizard>
<button (click)="opened=true" class="btn btn-primary">Show Wizard</button>

```

```
@Component({
  selector: "my-app",
  templateUrl: "./app.component.html",
  styleUrls: ["./app.component.css"],
  providers: [ClrAriaLiveService]
})
export class AppComponent {
  opened = false;
  loading = false;

  constructor(private clrAriaLiveService: ClrAriaLiveService) {}

  toggleLoading() {
    this.loading = !this.loading;
    if (this.loading) {
      console.log("Announcing...");
      this.clrAriaLiveService.announce("Loading", ClrAriaLivePoliteness.assertive);
    }
  }
}
```

**Comment by bdryanovski** [comment]
In my initial test, I could confirm that is not announcing the message and also is not removed from the DOM when the wizard is closed.

* for one we need to provide a method for manually removing the aria-live container when components are not destroyed when we expect (for example content-projection components)
* Need to find a way to force the aria-live to work with identical messages - something that is not always done by the scree-reader.

This is something that we talk internally, that we need to provide some additional methods and better documentation on use-cases - so this is a good candidate for improving on that. 

I will try to find some time to see what I could find more about the issue this week.

**Comment by mathisscott** [comment]
I dug into this issue some more. The problem is with how screen readers recognize dynamic content and also with how screen readers register, for lack of a better word, live regions.

I've been able to verify the ClrAriaLiveService doesn't work. Maybe it works in some isolated use cases?

We need to discuss a plan to fix this or offer a separate pattern that people can use in their applications so that way we can deprecate and remove this service.

**Comment by mathisscott** [comment]
Per a discussion with the team, the `ClrAriaLiveService` will be deprecated in v5 and removed in v6. We do not recommend using it in its present state.

For a workaround, we do recommend the Angular Material CDK LiveAnnouncer. More info on that can be found here: https://material.angular.io/cdk/a11y/overview#example-1

Whether similar functionality will be introduced as a service in `@clr/core` or not is still up for debate.
